e2e/common/utils.go

```diff
@@ -0,0 +1,193 @@
+// Copyright © 2021 Cisco Systems, Inc. and its affiliates.
+// All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package common
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"time"
+
+	v1 "k8s.io/api/core/v1"
+	v12 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/e2e-framework/klient"
+	"sigs.k8s.io/e2e-framework/third_party/helm"
+)
+
+// EXPORTED:
+
+func Deploy(ns string, yamlFile string) error {
+	cmd := exec.Command("kubectl", "-n", ns, "apply", "-f", yamlFile)
+
+	out, err := cmd.CombinedOutput()
+	if err != nil {
+		return fmt.Errorf("failed to execute command. %v, %s", err, out)
+	}
+	return nil
+}
+
+func LoadDockerImagesToCluster(cluster, tag string) error {
```

Comment:
maybe in each error worth saying what image failed (maybe adding the image names to consts will help)